sp800-63-3/definitions.md

```diff
@@ -44,28 +44,28 @@ A party, including an insider, who acts with malicious intent to compromise a sy
 A quality or characteristic ascribed to someone or something.
 
 #### Attribute Bundle
-A packaged set of attributes, usually contained within an assertion. Attribute bundles offer relying parties a simple way to retrieve the most relevant attributes they need from identity providers. Attribute bundles are synonymous with OpenID Connect scopes. See [OpenID Connect Core 1.0](http://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims).
+A packaged set of attributes, usually contained within an assertion. Attribute bundles offer RPs a simple way to retrieve the most relevant attributes they need from IdPs. Attribute bundles are synonymous with OpenID Connect scopes. See [OpenID Connect Core 1.0](http://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims).
```

Comment:
add open id connect core to the references.  Then the text would be [Open ID Connect core 1.0]